fast64_internal/render_settings.py

```diff
@@ -0,0 +1,160 @@
+import bpy
+import mathutils
+from .utility import get_blender_to_game_scale, transform_mtx_blender_to_n64
+
+
+def on_update_sm64_render_settings(self, context: bpy.types.Context):
+    renderSettings: "Fast64RenderSettings_Properties" = context.scene.fast64.renderSettings
+    if renderSettings.sm64Area and renderSettings.useObjectRenderPreview:
+        area: bpy.types.Object = renderSettings.sm64Area
+        renderSettings.fogPreviewColor = tuple(c for c in area.area_fog_color)
+        renderSettings.fogPreviewPosition = tuple(round(p) for p in area.area_fog_position)
+
+        renderSettings.clippingPlanes = tuple(float(p) for p in area.clipPlanes)
+
+
+def on_update_oot_render_settings(self, context: bpy.types.Context):
+    # TODO: Update render properties from selected OOTLightProperty

```

Comment:
I remember this being discussed, was it ever implemented on a different branch or is it stil TODO?

Reply:
yeah sauraen's PR right here https://github.com/thecozies/fast64/pull/4
after this we can ask them to repoint to `fast64/V5` since its pretty essential